Explorer/Assets/DCL/Passport/Modules/EquippedItems_PassportModuleController.cs

```diff
@@ -0,0 +1,316 @@
+                },
+                actionOnRelease: loadingItemView =>
+                {
+                    loadingItemView.SetAsLoading(false);
+                    loadingItemView.gameObject.SetActive(false);
+                }
+            );
+
+            equippedItemsPool = new ObjectPool<EquippedItem_PassportFieldView>(
+                InstantiateEquippedItemPrefab,
+                defaultCapacity: EQUIPPED_ITEMS_POOL_DEFAULT_CAPACITY,
+                actionOnGet: equippedItemView =>
+                {
+                    equippedItemView.gameObject.SetActive(true);
+                    equippedItemView.gameObject.transform.SetAsFirstSibling();
+                },
+                actionOnRelease: equippedItemView =>
+                {
+                    equippedItemView.gameObject.SetActive(false);
+                    equippedItemView.BuyButton.onClick.RemoveAllListeners();
+                });
+
+            emptyItemsPool = new ObjectPool<EquippedItem_PassportFieldView>(
+                InstantiateEquippedItemPrefab,
+                defaultCapacity: GRID_ITEMS_PER_ROW - 1,
+                actionOnGet: emptyItemView =>
+                {
+                    emptyItemView.gameObject.SetActive(true);
+                    emptyItemView.SetInvisible(true);
+                    emptyItemView.gameObject.transform.SetAsFirstSibling();
+                },
+                actionOnRelease: emptyItemView => emptyItemView.gameObject.SetActive(false));
+        }
+
+        public void Setup(Profile profile)
+        {
+            currentProfile = profile;
+
+            LoadEquippedItems();
+        }
+
+        public void Clear()
+        {
+            ClearLoadingItems();
+            ClearEquippedItems();
+            ClearEmptyItems();
+        }
+
+        public void Dispose() =>
+            Clear();
+
+        private EquippedItem_PassportFieldView InstantiateEquippedItemPrefab()
+        {
+            EquippedItem_PassportFieldView equippedItemView = Object.Instantiate(view.equippedItemPrefab, view.EquippedItemsContainer);
+            return equippedItemView;
+        }
+
+        private void LoadEquippedItems()
+        {
+            Clear();
+            SetGridAsLoading();
+
+            WearablePromise equippedWearablesPromise = WearablePromise.Create(
+                world,
+                WearableComponentsUtils.CreateGetWearablesByPointersIntention(currentProfile.Avatar.BodyShape, currentProfile.Avatar.Wearables, currentProfile.Avatar.ForceRender),
+                PartitionComponent.TOP_PRIORITY);
+
+            EmotePromise equippedEmotesPromise = EmotePromise.Create(
+                world,
+                EmoteComponentsUtils.CreateGetEmotesByPointersIntention(currentProfile.Avatar.BodyShape, currentProfile.Avatar.Emotes),
+                PartitionComponent.TOP_PRIORITY);
+
+            AwaitEquippedItemsPromiseAsync(equippedWearablesPromise, equippedEmotesPromise, cts.Token).Forget();
```

Comment:
Please check everywhere how you handle cancellations: in the most cases we are not interested in them, therefore they should be suppressed

Reply:
All unitasks reviewed (doing nothing for cancelled exceptions) and all "non-cancelled" exceptions managed by the new `PassportErrorsController` 👍 
Done here: https://github.com/decentraland/unity-explorer/pull/1277/commits/35023bc04a31075cbee07dfd52ba07f16892c3be